paloesg/excide
# Disable indexing algolia when run testing

# Description

- Disable indexing algolia when run testing

Trello link: https://trello.com/c/40Hxmc74

## Remarks

- none

# Testing

- Check the number of records in Algolia (Workflows or Documents)
- Run rspec, the number of records should not change when we run testing

## Checklist:
